bin/run-markdownlint-tests.sh

```diff
@@ -6,7 +6,7 @@ MARKDOWN_TEST_FILES=$(find "${SCRIPT_DIR}/../tests/markdownlint" -type f -name '
 
 TEST_DID_FAIL=0
 for TEST_FILE in $MARKDOWN_TEST_FILES; do
-   OUTPUT=$("${SCRIPT_DIR}/../node_modules/.bin/markdownlint" -c "${SCRIPT_DIR}/../.markdownlint.json" "${TEST_FILE}" 2>&1)
+   OUTPUT=$("${SCRIPT_DIR}/../node_modules/.bin/markdownlint-cli2-config" "${SCRIPT_DIR}/../.markdownlint-cli2.jsonc" "${TEST_FILE}" 2>&1)
```

Comment:
If we go with the shared config, we'll need something like the following code in this script:

```bash
# markdownlint-cli2 config files are merged with other versions of this file in a parent
# directory, up to the current directory. To prevent the test-specific config from being
# merged with the project's general config, we have to `cd` to the test directory.
# Docs: https://github.com/DavidAnson/markdownlint-cli2/tree/main#markdownlint-cli2jsonc
pushd "${SCRIPT_DIR}/../tests/markdownlint" > /dev/null
```